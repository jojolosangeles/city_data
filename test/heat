set -x
python ../code/main.py scripts/heat.txt > actual/heat.out
diff expected/heat.out actual/heat.out
