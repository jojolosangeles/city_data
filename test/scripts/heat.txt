all_areas = expected/all_areas.csv
all_areas.heat
 Hour
 Weekday
all_areas.heat
 Hour
 Victim Sex
all_areas.Age = [("<25" if (x < 25) else "<35" if (x < 35) else "<45" if (x < 45) else "<55" if (x < 55) else "<65" if (x < 65) else "<75" if (x < 75) else "Old") for x in dataFrame['Victim Age']]
all_areas.heat
 Age.heat
 Weekday
Northeast = expected/all_areas.Area_Name.Northeast.csv
Northeast.heat
 Hour
 Weekday
Northeast.heat
 Hour
 Victim Sex
