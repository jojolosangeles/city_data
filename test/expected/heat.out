Test: scripts/heat.txt
> load expected/all_areas.csv as all_areas
Loaded expected/all_areas.csv into DataFrame all_areas
> heat all_areas Hour Weekday
HeatMapCommand(all_areas) Hour, Weekday
Saved PNG heatmap: all_areas.Hour.Weekday.heatmap.png
> heat all_areas Hour Victim+Sex
HeatMapCommand(all_areas) Hour, Victim Sex
Saved PNG heatmap: all_areas.Hour.Victim_Sex.heatmap.png
> create all_areas Age [("<25" if (x < 25) else "<35" if (x < 35) else "<45" if (x < 45) else "<55" if (x < 55) else "<65" if (x < 65) else "<75" if (x < 75) else "Old") for x in dataFrame['Victim Age']]
all_areas.Age=
> heat all_areas Age Weekday
HeatMapCommand(all_areas) Age, Weekday
Saved PNG heatmap: all_areas.Age.Weekday.heatmap.png
> load expected/all_areas.Area_Name.Northeast.csv as Northeast
Loaded expected/all_areas.Area_Name.Northeast.csv into DataFrame Northeast
> heat Northeast Hour Weekday
HeatMapCommand(Northeast) Hour, Weekday
Saved PNG heatmap: Northeast.Hour.Weekday.heatmap.png
> heat Northeast Hour Victim+Sex
HeatMapCommand(Northeast) Hour, Victim Sex
Saved PNG heatmap: Northeast.Hour.Victim_Sex.heatmap.png
Test completed: scripts/heat.txt
